frontend/src/components/listofproducts/DeleteProductButton.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Button, Modal } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next/'
+import DeletionResponseModal from '../listofusers/DeletionResponseModal'
```

Comment:
Ez a modal mehetne valamiféle közös mappába szerintem, ha a usernél van, de a terméknél is használod.